WcaOnRails/db/migrate/20180825114051_clear_null_registration_requirements.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ClearNullRegistrationRequirements < ActiveRecord::Migration[5.2]
+  def change
+    Competition.visible.where(registration_requirements: "#").update_all(registration_requirements: nil)
```

Comment:
I would expect this to be `extra_registration_requirements` now that we've merged up https://github.com/thewca/worldcubeassociation.org/pull/3265.

Reply:
You are totally right!